pimd/pim_cmd_common.c

```diff
@@ -916,8 +912,8 @@ void pim_show_rpf(struct pim_instance *pim, struct vty *vty, json_object *json)
 			json_object_string_add(json_row, "group", grp_str);
 			json_object_string_add(json_row, "rpfInterface",
 					       rpf_ifname);
-			json_object_string_add(json_row, "rpfAddress",
-					       rpf_addr_str);
+			json_object_string_addf(json_row, "rpfAddress", "%pPAs",
```

Comment:
Inconsistent format for `rpfAddress`, you use `%pPAs` here and `%pPA` below :)

`%pPA` is better here, `%pPAs` is for multicast S to replace 0.0.0.0 with `*` - does not apply for RPF address.

Reply:
Fixed